brave-instrumentation/aws-java-sdk-core/src/main/java/brave/instrumentation/aws/AwsClientTracing.java

```diff
@@ -0,0 +1,55 @@
+package brave.instrumentation.aws;
+
+import brave.http.HttpTracing;
+import com.amazonaws.client.builder.AwsAsyncClientBuilder;
+import com.amazonaws.client.builder.AwsClientBuilder;
+import com.amazonaws.client.builder.ExecutorFactory;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.ForkJoinPool;
+
+public final class AwsClientTracing {
```

Comment:
I think similar to other types like this, we'd want to treat it as a component. Ex make HttpTracing a field (we might need others later). Then instead of a static method build, maybe an instance method buildClient()